addons/payment_razorpay/models/payment_transaction.py

```diff
@@ -49,6 +50,19 @@ def _get_specific_rendering_values(self, processing_values):
         converted_amount = payment_utils.to_minor_currency_units(self.amount, self.currency_id)
         base_url = self.provider_id.get_base_url()
         return_url_params = {'reference': self.reference}
+
+        phone = self.partner_phone
+        error_message = _("The phone number is missing.")
+        if phone:
+            # sanitize partner phone
+            country_code = self.partner_country_id.code
+            country_phone_code = self.partner_country_id.phone_code
+            phone_info = phone_sanitize_numbers([phone], country_code, country_phone_code)
+            phone = phone_info.get(phone, {}).get('sanitized')
+            error_message = phone_info.get(phone, {}).get('msg')
```

Comment:
Unlike the other place I asked for a change, here, we are changing phone to the sanitized phone, so we keep `self.partner_phone`, otherwise you won't get the correct key for the dictionary. _(Well, first line would work, but it's better to be consistent)_
```suggestion
            phone = phone_info.get(self.partner_phone, {}).get('sanitized')
            error_message = phone_info.get(self.partner_phone, {}).get('msg')
```